implants/golem/src/main.rs

```diff
@@ -10,13 +11,65 @@ use eldritch::{eldritch_run};
 
 mod inter;
 
+#[cfg(debug_assertions)]
+#[derive(RustEmbed)]
+#[folder = "../../tests/embedded_files_test"]
+pub struct Asset;
+
+#[cfg(not(debug_assertions))]
+#[derive(RustEmbed)]
+#[folder = "../../implants/golem/embed_files_golem_prod"]
+pub struct Asset;
```

Comment:
I would like to find a better way to define the assets structure.
Currently it gets defined for a test run & and a build.
It also gets defined here in eldritch and in golem.

Reply:
We can pass it to the interpreter to help solve this problem